### PR TITLE
feat(integer): add trim/extend APIs for radix ciphertexts

### PR DESCRIPTION
**Pull request by IceTDrinker**

### CODE REVIEW

File: tfhe/src/integer/server_key/radix/mod.rs [hunk 1]
```diff
@@ -88,6 +88,319 @@ impl ServerKey {
...
+    /// ```
+    pub fn extend_radix_with_trivial_zero_blocks_msb_assign<PBSOrder>(
+        &self,
+        ct: &mut RadixCiphertext<PBSOrder>,
+        num_blocks: usize,
+    ) where
+        PBSOrder: PBSOrderMarker,
+    {
+        let block_trivial_zero = self.key.create_trivial(0);
+        ct.blocks
```
Comment (tmontaigu) [inline tfhe/src/integer/server_key/radix/mod.rs:209]: Vec::resize or Vec::resize_with could be have been used instead of iterators

Comment (IceTDrinker) [reply tfhe/src/integer/server_key/radix/mod.rs:209]: indeed, not sure what I was thinking

File: tfhe/src/integer/server_key/radix/mod.rs [hunk 2]
```diff
@@ -88,6 +88,319 @@ impl ServerKey {
         )
     }
 
+    /// Add trivial zero LSB blocks to an existing [`RadixCiphertext`]. This can be useful for
```
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/src/integer/server_key/radix/mod.rs]: done

File: tfhe/src/integer/server_key/radix/mod.rs (hunk 1 above)
Comment (IceTDrinker) [reply tfhe/src/integer/server_key/radix/mod.rs:209]: done

**Review by tmontaigu — APPROVED** [review]

**Pull request merged**